server/middleware/checkServiceEnabledForActiveCaseLoad.ts

```diff
@@ -19,24 +14,9 @@ export default function checkPopulateUserCaseloads(
 
   router.use(async (req, res, next) => {
     const splitUrl = req.url.split('/').filter(Boolean)
-    const services = res.locals.feComponentsMeta?.services
+    const services = res.locals.feComponents?.sharedData?.services
     let isEligibleForService = services ? services.find(service => service.id === 'csipUI') !== undefined : undefined
-    let caseloads
     try {
-      if (res.locals.feComponentsMeta?.caseLoads) {
-        res.locals.user.caseloads = res.locals.feComponentsMeta.caseLoads
-      }
-      if (res.locals.feComponentsMeta?.activeCaseLoad) {
-        res.locals.user.activeCaseLoad = res.locals.feComponentsMeta.activeCaseLoad
-      }
-      const refetchCaseloads = !res.locals.user.caseloads
-      if (refetchCaseloads) {
-        caseloads = caseloads ?? (await prisonApiService.getCaseLoads(req))
-        res.locals.user.caseloads = caseloads as CaseLoad[]
-        res.locals.user.activeCaseLoad =
-          (caseloads as CaseLoad[])!.find(caseload => caseload.currentlyActive) ?? res.locals.user.activeCaseLoad
-      }
-
       // Check that the user's active caseload is enabled on the API side
```

Comment:
All that's left here is the bespoke check on active case load being supported by the service...